examples/games/padracing/lib/car.dart

```diff
@@ -102,15 +102,15 @@ class Car extends BodyComponent<PadRacingGame> {
       final isLeftTire = i.isEven;
       return Tire(
         car: this,
-        pressedKeys: gameRef.pressedKeySets[playerNumber],
+        pressedKeys: game.pressedKeySets[playerNumber],
```

Comment:
why the rename from gameRef to game?

Reply:
That is from the really old Pasha PR where we are moving from the `HasGameRef` mixin to `HasGameReference`.